sklearn/tests/test_metaestimators.py

```diff
@@ -197,6 +198,52 @@ def score(self, X, y, *args, **kwargs):
             )
 
 
+def _get_instance_with_pipeline(meta_estimator, init_params):
```

Comment:
Uhm codecov is complaining of not covered line here. I would not expect it since this should be some code that we have before.

Reply:
It isn't complaining anymore, I think.